modules/azure-gaming/game-dev-vmss/main.bicep

```diff
@@ -0,0 +1,247 @@
+fileShareStorageAccountKey={1}
+fileShareName={2}
+p4Port={3}
+p4Username={4}
+p4Password={5}
+p4Workspace={6}
+p4Stream={7}
+p4ClientViews={8}
+ibLicenseKey={9}
+gdkVersion={10}
+useVmToSysprepCustomImage={11}
+remoteAccessTechnology={12}
+teradiciRegKey={13}
+parsecTeamId={14}
+parsecTeamKey={15}
+parsecHost={16}
+parsecUserEmail={17}
+parsecIsGuestAccess={18}
+deployedFromSolutionTemplate={19}
+''', fileShareStorageAccount, fileShareStorageAccountKey, fileShareName, p4Port, p4Username, p4Password, p4Workspace, p4Stream, p4ClientViews, ibLicenseKey, gdkVersion, useVmToSysprepCustomImage, remoteAccessTechnology, teradiciRegKey, parsecTeamId, parsecTeamKey, parsecHost, parsecUserEmail, parsecIsGuestAccess, false)
+
+
+module vnet './nestedtemplates/virtualNetworks.bicep'  = {
+  name:                       vnetName
+  params: {
+    location:                 location
+    vnetName:                 vnetName
+    subnetName:               subnetName
+    networkSecurityGroupName: networkSecurityGroupName
+    vnetAddressPrefix:        vnetAddressPrefix
+    subnetAddressPrefix:      subnetAddressPrefix
+  }
+}
+
+resource vmss 'Microsoft.Compute/virtualMachineScaleSets@2021-04-01' = {
+  name: vmssName
+  location: location
+  sku: {
+    name:     vmssSku
+    tier:     'Standard'
+    capacity: vmssInstanceCount
+  }
+  plan: {
+    name:      vmssImgSku
+    publisher: vmssImgPublisher
+    product:   vmssImgProduct
+  }  
+  properties: {
+    singlePlacementGroup: false
+    upgradePolicy: {
+      mode: 'Manual'
+    }
+    virtualMachineProfile: {
+      storageProfile: {
+        osDisk: {
+          createOption: 'FromImage'
+          caching: 'ReadWrite'
+          managedDisk: {
+            storageAccountType: vmssOsDiskType
+          }
+        }
+        imageReference: {
+          publisher: vmssImgPublisher
+          offer:     vmssImgProduct
+          sku:       vmssImgSku
+          version:   vmssImgVersion
+	}
+      }
+      networkProfile: {
+        networkInterfaceConfigurations: [
+          {
+            name: '${vmssName}Nic'
+            properties: {
+              primary: true
+              ipConfigurations: [
+                {
+                  name: '${vmssName}IpConfig'
+                  properties: {
+                    subnet: {
+                      id: vnet.outputs.subnetId
+                    }
+                  }
+                }
+              ]
+              networkSecurityGroup: {
+                id: vnet.outputs.nsgID
+              }
+            }
+          }
+        ]
+      }
+      osProfile: {
+        computerNamePrefix: vmssName
+        adminUsername:      administratorLogin
+        adminPassword:      passwordAdministratorLogin
+	customData:         base64(customData)
+	windowsConfiguration: {
+          provisionVMAgent: true
+        }	
+      }
+      priority: 'Regular'
+    }
+    overprovision: false
+  }
+}
+
+@description('VMSS ID')
+output id   string = vmss.id
```

Comment:
remove extra whitespace between `id` and `string`

Reply:
```suggestion
output id string = vmss.id
```